notebooks/image_models/solutions/object_detection_automl_vertex.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        print(f"Intersection over union (IoU): {metric['iouThreshold']}\n")
</pre><p>Can you add a line in the markdown above is what IoU threshold means?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>